Packs/AWSCompromisedEC2Response/Playbooks/playbook-AWS_Compromised_EC2_Response.yml

```diff
@@ -0,0 +1,618 @@
+          "x": -230,
+          "y": 1070
+        }
+      }
+  "11":
+    id: "11"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "10"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      availabilityZone:
+        complex:
+          accessor: AvailabilityZone
+          root: AWS.EC2.Instances.Placement
+      encrypted: {}
+      iops: {}
+      kmsKeyId: {}
+      region: {}
+      roleArn: {}
+      roleSessionDuration: {}
+      roleSessionName: {}
+      size: {}
+      snapshotId:
+        complex:
+          accessor: SnapshotId
+          root: AWS.EC2.Snapshots
+      tags: {}
+      volumeType: {}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: AWS - EC2
+      description: Creates an EBS volume that can be attached to an instance in the
+        same Availability Zone.
+      id: 71935b73-4ac5-4d7d-8009-46286417668f
+      iscommand: true
+      name: Convert Snapshot to Volume
+      script: AWS - EC2|||aws-ec2-create-volume
+      type: regular
+      version: -1
+    taskid: 71935b73-4ac5-4d7d-8009-46286417668f
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": -230,
+          "y": 895
+        }
+      }
+  "12":
+    id: "12"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "1"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      filter: {}
+      instanceIds:
+        complex:
+          accessor: InstanceId
+          root: AWS.EC2.Instances.[1]
+      region: {}
+      roleArn: {}
+      roleSessionDuration: {}
+      roleSessionName: {}
+      waiterDelay: {}
+      waiterMaxAttempts: {}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: AWS - EC2
+      description: A waiter function that runs every 15  seconds until a successful
+        state is reached.
+      id: 4d78360c-4262-4a90-8d5c-fd30993cfea6
+      iscommand: true
+      name: Wait for Forensic Instance Initialization
+      script: AWS - EC2|||aws-ec2-waiter-instance-running
+      type: regular
+      version: -1
+    taskid: 4d78360c-4262-4a90-8d5c-fd30993cfea6
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": -230,
+          "y": 1245
+        }
+      }
+  "15":
+    id: "15"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "16"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      CidrBlock:
+        complex:
+          accessor: ipAddressV4
+          root: incident.service.action.networkConnectionAction.remoteIpDetails
+          transformers:
+          - args:
+              prefix: {}
+              suffix:
+                value:
+                  simple: /32
+            operator: concat
+      Code: {}
+      DryRun: {}
+      Egress:
+        simple: "True"
+      From:
+        simple: "0"
+      Ipv6CidrBlock: {}
+      NetworkAclId:
+        simple: acl-0780ee09f0dbd6f7c
```

Comment:
Please replace the hard-coded `NetworkAclId` input with playbook input / some other non hard-coded input.